pro/src/screens/OfferIndividual/StocksEventCreation/form/__specs__/onSubmit.spec.ts

```diff
@@ -51,6 +52,43 @@ describe('onSubmit', () => {
         },
       ],
     },
+    {
+      description: 'on a daily basis',
```

Comment:
peut-être que la description pourrait contenir l'intitulé complet du test ?  les tests sont un peu difficiles à lire sous ce format, on ne sait pas où ça commence, ça aiderait à structurer, par exemple `should create stocks on a daily basis` (en enlevant `should validate the form for case` plus bas, en réalité on ne teste pas vraiment la validation dans le expect, mais que les stocks générés sont corrects)